commcare_connect/opportunity/views.py

```diff
@@ -813,3 +826,31 @@ def update_completed_work_status_import(request, org_slug=None, pk=None):
             message += status.get_missing_message()
         messages.success(request, mark_safe(message))
     return redirect("opportunity:detail", org_slug, pk)
+
+
+@org_member_required
+@require_POST
+def suspend_user(request, org_slug=None, opp_id=None, pk=None):
+    access = get_object_or_404(OpportunityAccess, opportunity_id=opp_id, id=pk)
+    access.suspended = True
+    access.suspension_date = now()
+    access.suspension_reason = request.POST.get("reason", "")
+    access.save()
```

Comment:
What happens to existing accrued and approved payments when user is suspended? Shouldn't they be rejected here?

Reply:
The existing accrued and approved payments are not touched when a user is suspended. The Spec mentioned that we should not make changes to data that is already on the platform and only focus on the data coming after the user is suspended.

Reply:
Cool.